From b7945a50cb7aa088ed1cd71d8487e7fc0c3d6322 Mon Sep 17 00:00:00 2001
From: Juan Cruz Viotti <jv@jviotti.com>
Date: Fri, 16 Feb 2024 12:36:52 -0400
Subject: [PATCH] Disable ENABLE_API_TESTS

Signed-off-by: Juan Cruz Viotti <jv@jviotti.com>
---
 Source/cmake/OptionsJSCOnly.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Source/cmake/OptionsJSCOnly.cmake b/Source/cmake/OptionsJSCOnly.cmake
index e892b42..2190967 100644
--- a/Source/cmake/OptionsJSCOnly.cmake
+++ b/Source/cmake/OptionsJSCOnly.cmake
@@ -47,7 +47,7 @@ set(ENABLE_WEBGL OFF)
 if (WIN32)
     set(ENABLE_API_TESTS OFF)
 else ()
-    set(ENABLE_API_TESTS ON)
+    set(ENABLE_API_TESTS OFF)
 endif ()
 
 if (WTF_CPU_ARM OR WTF_CPU_MIPS)
-- 
2.43.1

